core/src/main/scala/org/ensime/indexer/SearchService.scala

```diff
@@ -219,22 +220,13 @@ class SearchService(
         val source = resolver.resolve(clazz.name.pack, clazz.source)
         val sourceUri = source.map(_.getName.getURI)
 
+        val sourceFileInfo = (name, path, sourceUri)
         if (clazz.access != Public) Nil
         else {
-          FqnSymbol(None, name, path, clazz.name.fqnString, None, sourceUri, clazz.source.line) ::
-            clazz.methods.toList.filter(_.access == Public).map { method =>
-              FqnSymbol(None, name, path, method.name.fqnString, None, sourceUri, method.line)
-            } ::: clazz.fields.toList.filter(_.access == Public).map { field =>
-              val internal = field.clazz.internalString
-              FqnSymbol(None, name, path, field.name.fqnString, Some(internal), sourceUri, clazz.source.line)
-            } ::: depickler.getTypeAliases.toList.filter(_.access == Public).filterNot(_.fqn.contains("<refinement>")).map { rawType =>
-              // this is a hack, we shouldn't be storing Scala names in the JVM name space
-              // in particular, it creates fqn names that clash with the above ones
-              FqnSymbol(None, name, path, rawType.fqn, None, sourceUri, None)
-            }
+          (clazz :: clazz.methods.toList ::: clazz.fields.toList ::: depickler.getTypeAliases.toList).map(_ -> sourceFileInfo)
```

Comment:
Why are these fields `Queue`s in the first place? We never seem to use FIFO nature of them anyways.